### PR TITLE
Saving

### PR DESCRIPTION
**Pull request by vaa25**
- ExcelCellRange not supported in writing.
- ExcelCellName has new parameter 'order'. It used to set column number for field.
- Xlsx support huge data.
- support thread-safe saving 
- Fix Boolean, byte, Byte, short, Short casting
- Add customizing of CellCasting (field to cell casting for toExcel())

### CODE REVIEW

**Comment by ozlerhakan** [comment]
hey @vaa25 , thanks for the changes, we need to improve the coverage first  for the pr

**Comment by vaa25** [comment]
@ozlerhakan feature looks completed. Review please.

File: pom.xml [hunk 1]
```diff
@@ -70,6 +70,12 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (ozlerhakan) [inline pom.xml]: we need to keep our dependencies simpler, so lombok should be removed.

**Comment by ozlerhakan** [comment]
thank you for your effort @vaa25 , since this is a big change for poiji, I need to look at them closely however, currently I don't have time to do that :/ Let's keep it here for now

**Comment by vaa25** [comment]
moved to https://github.com/vaa25/poiji

**Pull request closed without merging**